content/blog/2017-12-07-introducing-the-react-rfc-process.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Introducing the React RFC Process"
+author: [acdlite]
+---
+
+We're adopting an RFC ("request for comments") process for contributing ideas to React. 
+
+Inspired by [Yarn](https://github.com/yarnpkg/rfcs), [Ember](https://github.com/emberjs/rfcs), and [Rust](https://github.com/rust-lang/rfcs), the goal is to allow React core team members and community members to collaborate on the design of new features. It's also intended to provide a clear path for ideas to enter the project:
+
+- Create an RFC document detailing your proposal.
+- Submit a PR to the [RFC repository](https://github.com/reactjs/rfcs).
+- Incorporate feedback into the proposal.
+- After discussion, the RFC may or may not be accepted by the core team.
+- If the RFC is accepted, the PR is merged.
+
+RFCs are accepted when they are approved for implementation in React. A more thorough description of the process is available in the repository's [README](https://github.com/reactjs/rfcs/blob/master/README.md). The exact details may be refined in the future.
+
+## Who Can Submit RFCs?
+
+Anyone! No knowledge of React's internals is required, nor are you expected to implement the proposal yourself.
+
+As with our other repositories, we do ask that you complete a [Contributor License Agreement](https://github.com/reactjs/rfcs#contributor-license-agreement-cla) before we can accept your PR.
+
+## What Types of Changes Should Be Submitted As RFCs?
+
+Generally, any idea that would benefit from additional review or design before being implemented is a good candidate for an RFC. As a rule of thumb, this means any proposal that adds, changes, or removes a React API.
+
+Not every change must go through the RFC process. Bug fixes or performance improvements that don't touch the API can be submitted directly to the main library.
+
+We now have several different repositories for contributing improvements to React:
```

Comment:
The meaning of this sentence is a little unclear to me. Do those repositories contribute improvements to React? Or are you able to contribute to React by contributing to of the mentioned repositories? I'm leaning towards the latter, but it took a few re-reads.

Reply:
Not sure if the sentence would have stood out to me as much before seeing Clam's comment, but now it does.